cex-collection/js-example/.env

```diff
@@ -1,10 +1,10 @@
 #PAYMASTER_URL="https://bsc-megafuel.nodereal.io"
-#SPONSOR_URL="https://open-platform-ap.nodereal.io/{YOUR_API_KEY}/megafuel"
-
+#SPONSOR_URL="https://open-platform-ap.nodereal.io/{YOUR_API_KEY}/megafuel/56"
 PAYMASTER_URL="https://bsc-megafuel-testnet.nodereal.io"
-SPONSOR_URL="https://open-platform-ap.nodereal.io/{YOUR_API_KEY}/megafuel-testnet"
+SPONSOR_URL="https://open-platform-ap.nodereal.io/{YOUR_API_KEY}/megafuel-testnet/97"
 
 POLICY_UUID="xxxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxx"
 TOKEN_CONTRACT_ADDRESS="0xeD24F....12Ee"
 CONSOLIDATION_WALLET_ADDRESS="0x8e92....3EA2"
-DEPOSIT_WALLET_PRIVATE_KEY="69...929"
\ No newline at end of file
+DEPOSIT_WALLET_PRIVATE_KEY="69...929"
+CHAIN_ID="97"
```

Comment:
please simplify it so that we dont have to have `CHAIN_ID` here, as go example dont have it.